proxy/src/main/java/net/md_5/bungee/connection/InitialHandler.java

```diff
@@ -492,11 +491,20 @@ public void run()
                             {
                                 server = AbstractReconnectHandler.getForcedHost( InitialHandler.this );
                             }
+
                             if ( server == null )
                             {
                                 server = bungee.getServerInfo( listener.getDefaultServer() );
                             }
 
+                            PostLoginEvent postLoginEvent = new PostLoginEvent( userCon );
+                            postLoginEvent.setTarget(server);
+                            postLoginEvent = bungee.getPluginManager().callEvent( postLoginEvent  );
```

Comment:
If you want to remove that one space at the end... :D
Then it looks perfect to be merged by the md5 or thinkofdeath